extensions/metrics-cluster-feature/src/metrics-settings.tsx

```diff
@@ -164,13 +187,13 @@ export class MetricsSettings extends React.Component<Props> {
           <Component.FormSwitch
             control={
               <Component.Switcher
-                disabled={this.featureStates.kubeStateMetrics === undefined || !this.props.cluster.status.active}
+                disabled={this.featureStates.kubeStateMetrics === undefined || !this.isTogglable}
                 checked={!!this.featureStates.prometheus && this.props.cluster.status.active}
                 onChange={v => this.togglePrometheus(v.target.checked)}
                 name="prometheus"
               />
             }
-            label="Install built-in Prometheus metrics stack"
+            label="Install bundled Prometheus metrics stack"
```

Comment:
Since you changed "installed" to "enabled" above should this be
```suggestion
            label="Enable bundled Prometheus metrics stack"
```
and similarly below?

Or just
```suggestion
            label="Bundled Prometheus metrics stack"
```